.github/workflows/macos.yml

```diff
@@ -1,16 +1,24 @@
-name: C/C++ CI MacOS
+name: MacOS
 
 on:
   push:
-    branches: [ $default-branch ]
   pull_request:
     types: [ opened, reopened, synchronize ]
 
 jobs:
   builds:
-    runs-on: macos-12
+    strategy:
+      fail-fast: false
+        # ubuntu 16.04 EOL is 2026
+      matrix: 
+        os: [ 'macos-11', 'macos-12' ]
```

Comment:
Why adding 11?

Reply:
github supports MacOSX, although if you would rather not support it it can be removed

Reply:
IMHO coturn should explicitly declare the platforms supported, and out of that list, any that github has runners for should be used.

If Mac OSX 11 isn't supported, then lets declare that to be the case. Else, take advantage of the equipment offered by github.

Reply:
I think one version of mac is enough
Especially if that is older one